.github/workflows/build-test.yml

```diff
@@ -47,3 +47,13 @@ jobs:
 
     - name: Test
       run: cargo test --verbose
+
+  format:
```

Comment:
What is the best practice with regard to adding a new yml file versus adding a section to the existing yml file?